src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperDescriptorFactory.cs

```diff
@@ -388,9 +394,24 @@ private static bool ShouldSkipDescriptorCreation(bool designTime, MemberInfo mem
             return false;
         }
 
+        private static bool ShouldSkipDescriptorCreation(bool designTime, IPropertyInfo property)
```

Comment:
Makes me feel like we should have a common interface of the two `ITypeInfo` and `IPropertyInfo` (much like `MemberInfo`). Today `EditorBrowsableAttribute` is the only one that applies to both classes and properties but it'd be nice to have the foundation laid out for other similar properties (since we already have one anyways that'd use it).